src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/Interfaces/IVsTestConsoleWrapper.cs

```diff
@@ -119,6 +120,14 @@ public interface IVsTestConsoleWrapper
         /// <param name="customTestHostLauncher">Custom test host launcher for the run.</param>
         void RunTestsWithCustomTestHost(IEnumerable<TestCase> testCases, string runSettings, TestPlatformOptions options, ITestRunEventsHandler testRunEventsHandler, ITestHostLauncher customTestHostLauncher);
 
+        /// <summary>
+        /// Provides back all attachments to TestPlatform for additional processing (for example merging)
+        /// </summary>
+        /// <param name="attachments">Collection of attachments</param>
+        /// <param name="multiTestRunFinalizationCompleteEventsHandler">EventHandler to receive session complete event</param>
+        /// <param name="cancellationToken">Cancellation token</param>
+        Task FinalizeMultiTestRunAsync(ICollection<AttachmentSet> attachments, IMultiTestRunFinalizationEventsHandler multiTestRunFinalizationCompleteEventsHandler, CancellationToken cancellationToken);
```

Comment:
>IMultiTestRunFinalizationEventsHandler [](start = 79, length = 38)

Add telemetry bool and IEnumerable. Recheck handler to be in same project as handlers for test run

Reply:
added